lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+<div class="calc-main">
+    <a title="${_('Open Calculator')}" href="#" role="button" aria-controls="calculator_wrapper" aria-expanded="false" class="calc">${_("Open Calculator")}</a>
```

Comment:
Cool, I'll ``<button``> that fella up in that case and will have @cptvitamin review a sandbox to make sure the interaction is solid.